src/platform/backends/lxd/lxd_vm_image_vault.cpp

```diff
@@ -187,6 +188,11 @@ mp::VMImage mp::LXDVMImageVault::fetch_image(const FetchType& fetch_type, const
     {
         // Instance doesn't exist, so move on
     }
+    catch (const LocalSocketConnectionException& e)
+    {
+        mpl::log(mpl::Level::warning, category, fmt::format("{} - returning blank image info", e.what()));
+        return VMImage{};
+    }
```

Comment:
> I'm unsure what catch-all further up you are referring to.

I meant the catch-all just below here, but your other answer in the other PR explains why that. We should definitely fix these to be more specific in the other image hosts at some point.